### PR TITLE
Add ruby and deb-s3 to CI image

### PR DESCRIPTION
**Pull request by umanwizard**

### CODE REVIEW

File: ci/builder/Dockerfile [hunk 1]
```diff
@@ -158,6 +159,10 @@ RUN pip3 install \
     -r /workdir/requirements-dev.txt \
     && rm /workdir/requirements*.txt
 
+RUN curl -fsSL https://github.com/deb-s3/deb-s3/releases/download/0.11.3/deb-s3-0.11.3.gem > deb-s3-0.11.3.gem \
+    && gem install ./deb-s3-0.11.3.gem
+
```
Comment (quodlibetor) [inline ci/builder/Dockerfile:164]: These two RUN commands should be merged, each separate one adds a small amount of time to docker image download/start time.

```suggestion
    -r /workdir/requirements-dev.txt \
    && rm /workdir/requirements*.txt \
    && curl -fsSL https://github.com/deb-s3/deb-s3/releases/download/0.11.3/deb-s3-0.11.3.gem > deb-s3-0.11.3.gem \
    && gem install ./deb-s3-0.11.3.gem

```

**Review by quodlibetor — COMMENTED** [review]
This commit message doesn't explain why you're adding this, and there are no code changes that make it obvious, so I'm not sure how to review whether this is correct.

**Comment by umanwizard** [comment]
Sorry, I forgot Nikhil was out when I put this up, and he already has context. I’ll add a note to the commit message, but the context is that we are going to use the deb-s3 gem to deploy apt repos to s3, so we need a Ruby interpreter as well as that gem in the builder image

**Review by quodlibetor — APPROVED** [review]

**Comment by quodlibetor** [comment]
That sounds reasonable to me.

**Comment by umanwizard** [comment]
I probably didn't need to bother updating the Nightly image, but no reason not to keep them in sync ¯\\_(ツ)_/¯

**Pull request merged**